test/images/dind-test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM        gliderlabs/alpine:3.2
```

Comment:
We probably could add the building of this image to https://github.com/taskcluster/docker-worker/blob/master/build.sh